### PR TITLE
multi-select export UAT changes

### PR DESCRIPTION
**Pull request by snopoke**
Also: https://github.com/dimagi/couchexport/pull/124

### CODE REVIEW

**Comment by millerdev** [comment]
g2g when tests pass

File: corehq/apps/reports/models.py [hunk 1]
```diff
@@ -644,6 +644,12 @@ class FormExportSchema(HQExportSchema):
     def update_schema(self):
         super(FormExportSchema, self).update_schema()
         self.update_question_schema()
+        # update SplitColumn options
+        for column in [column for table in self.tables for column in table.columns]:
+            if isinstance(column, SplitColumn):
+                question = self.question_schema.question_schema.get(column.index)
+                column.options = question.options
+                column.ignore_extras = True
```
Comment (millerdev) [inline corehq/apps/reports/models.py]: I'm questioning my earlier g2g comment.

I assume this column is local to `self` (i.e., not a global)? If not, seems like mutating it here could be problematic for the same reason modifying global state is ever questionable.

Comment (snopoke) [inline corehq/apps/reports/models.py]: column is local to this `FormExportSchema` object. This doesn't get saved to the DB either but even if it did it wouldn't matter.

Comment (millerdev) [inline corehq/apps/reports/models.py]: :+1:

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**